### PR TITLE
Task 3.3

### PR DESCRIPTION
**Pull request by shigontsev**

### CODE REVIEW

File: Task_3/Task_3.3/Task_3.3.1/NumberArrayExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+namespace Task_3._3._1
+{
+    public static class NumberArrayExtensions
+    {
+        #region Sbyte type
+        public static sbyte Sum(this sbyte[] arr)
+        {
+            arr.HelperExeption();
+
+            return (sbyte) arr.Sum(x => x);
```
Comment (STeeL835) [inline Task_3/Task_3.3/Task_3.3.1/NumberArrayExtensions.cs]: OverflowException - sum of signed bytes can be more than 1 signed byte (127+127 = 254, which is more than sbyte.MaxValue)

File: Task_3/Task_3.3/Task_3.3.1/NumberArrayExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+        private static void HelperExeption<T>(this T[] source)
+        {
+            if (source is null)
+            {
+                throw new ArgumentNullException("array is null");
+            }
+            if (source.Length == 0)
+            {
+                throw new ArgumentException("array is empty");
+            }
```
Comment (STeeL835) [inline Task_3/Task_3.3/Task_3.3.1/NumberArrayExtensions.cs]: You shouldn't throw an exception for an empty array. In the worst case nothing will enumerated

File: Task_3/Task_3.3/Task_3.3.3/Entity/Pizza.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    throw new ArgumentException(nameof(Name) + " is Empty");
+                }
+
+                _name = value; 
+            }
+        }
+
+        private TimeSpan _timeCook;
+
+        public TimeSpan TimeCook
```
Comment (STeeL835) [inline Task_3/Task_3.3/Task_3.3.3/Entity/Pizza.cs]: TimeToCook or CookingTime

File: Task_3/Task_3.3/Task_3.3.3/MakerOrders.cs [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                Console.Write("Enter number of pizza: ");
+                int.TryParse(Console.ReadLine().Trim(), out int select);
+                ChangePizza(select);
+                Console.Write("Enter any button for continue...");
+                Console.ReadKey();
+                Console.WriteLine(Environment.NewLine);
+            } while (true);
+        }
+
+        private void ChangePizza(int select)
```
Comment (STeeL835) [inline Task_3/Task_3.3/Task_3.3.3/MakerOrders.cs]: I think you meant OrderPizza

**Review by STeeL835 — APPROVED** [review]

**Comment by shigontsev** [comment]
End

**Pull request merged**